### ISSUE TITLE
Add inline preview like sublime LaTeXTools

### ISSUE THREAD

**Issue by wolfv** [jupyterlab/jupyterlab-latex#54]
E.g. https://imgur.com/Au4mbcF

**Comment by mpacer** [comment]
This is out of scope for a while, but not an unreasonable request. 

@ian-r-rose, do you concur?

**Comment by ian-r-rose** [comment]
I think it would be really cool, but there are a few things that would need to happen first.

I think it might be nice to plug this in to the completer machinery, so you could tab-to-preview an equation. There are already some plans to hook a completer up to a text editor, so if that were made sufficiently pluggable, we could try to use the client-side equation renderer in a hover box. 

Actually putting it inline would be fairly tricky to negotiate with the Codemirror instance, thought it may be possible.
